cluster_types.go

```diff
@@ -30,6 +30,8 @@ type Cluster struct {
 	ManagementAddress string `json:"management_address,omitempty"`
 	// Current state of the cluster
 	State string `json:"state,omitempty"`
+	// NVMe Subsystem NQN for cluster
```

Comment:
The comment should be  `NVMeNQN` right?

Reply:
Just added the usability similar to what is mentioned on the swagger, similar to the way mentioned for other parameters.